IDP custom reports docs (#33102)
DataDog/documentation: content/en/internal_developer_portal/eng_reports/custom_reports.md

---
title: Custom Reports
aliases:

> - /software_catalog/eng_reports/custom_reports/

Reviewer: @roxanne-moslehi Is this a link that existed before?

further_reading:
- link: "dashboards/"
  tag: "Documentation"
  text: "Datadog Dashboards"
---

{{< callout url="https://www.datadoghq.com/product-preview/idp-preview-features/" d_target="#signupModal" btn_hidden="false" header="Join the Preview for Custom Reports in IDP!" >}}
{{< /callout >}}

## Overview

> With Custom Reports, you can embed any Datadog Dashboard directly into the IDP Overview for all users at your organization. This makes it easy to customize the Overview Page so that different users can find the information they need directly in IDP, without context switching. 

Reviewer 2: just a suggestion to shorten the sentence

{{< img src="/tracing/eng_reports/idp-custom-reports.png" alt="Example custom report in IDP Overview page" style="width:100%;" >}}

## Managing Custom Reports

> Navigate to IDP > Overview, and you will see a “Custom Reports” section on the left navigation. Users with the [Software Catalog write permission][1], which is included in the default Datadog Admin Role and Datadog Standard Role, can manage Custom Reports. You can add, remove, or adjust the order of Datadog Dashboards that appear in the Custom Reports section. Note that Custom Reports are set at the organization level, so all users will see the same list of Custom Reports in the IDP Overview page. [Dashboard permissions][2] will also apply to any Dashboards added to IDP. 

Reviewer 2: ```suggestion
In IDP > Overview, see **Custom Reports**. 

Users with the [Software Catalog write permission][1], included in the default Datadog Admin Role and Datadog Standard Role, can manage Custom Reports. You can add, remove, or adjust the order of Datadog Dashboards that appear in the Custom Reports section. 

**Note:** Custom Reports are set at the organization level. All users in the org will see the same list of Custom Reports in the IDP Overview page. [Dashboard permissions][2] also apply to any Dashboards added to IDP. 
```

---
title: Custom Reports
aliases:

> - /software_catalog/eng_reports/custom_reports/

Reviewer: ```suggestion
```

further_reading:
- link: "dashboards/"
  tag: "Documentation"
  text: "Datadog Dashboards"
---

## Overview

Custom Reports let you embed existing Datadog dashboards directly into the IDP Overview page for all users at your organization. This allows teams to view key engineering insights in one place without context switching navigating between IDP and Dashboards.

{{< img src="/tracing/eng_reports/idp-custom-report.png" alt="Example custom report in IDP Overview page" style="width:100%;" >}}

## Viewing Custom Reports

To view Custom Reports, go to **IDP** > **Overview** and find the **Custom Reports** section. Custom Reports are set at the organization level, so all users see the same list. 

Select a Custom Report to view it directly in the IDP Overview page. You can adjust template variable filters and the time frame in place.

**Note**: Individual [dashboard permissions][2] apply to any dashboards added to IDP's Custom Reports. 

## Managing Custom Reports

Users with the [Software Catalog write permission][1], included in the default Datadog Admin and Datadog Standard Roles, can add, remove, or reorder the dashboards in the Custom Reports section. 

> To modify the content of a Custom Report or set up scheduled PDF reports through Slack or email, select **Edit Report**. This opens the underlying dashboard in the Dashboards UI. Edits made in the dashboard, such as changing widgets, updating queries, or adjusting template variables, automatically apply to the Custom Report in IDP. 

Reviewer 3: It's Manage Dashboard! Updating
